deploy/static-sites.txt

```diff
@@ -1,2 +1,3 @@
 documentation/**
-textbook/**
\ No newline at end of file
+textbook/**
+ecosystem/**
```

Comment:
I think this will exclude subdirectories, but not the files. Am I right @mtreinish ?
```suggestion
ecosystem/*/
```

Reply:
other option is to move to `--filter-from`, but not sure.